hack/check_tlsprofile.sh

```diff
@@ -21,26 +21,36 @@
 set -ex
 
 clean_nmap_output () {
-  sed -i '/^$/d' $1
-  sed -i '/^Starting Nmap.*$/d' $1
-  sed -i '/^Nmap scan report for.*$/d' $1
-  sed -i '/^Starting Nmap.*$/d' $1
-  sed -i '/^Host is up.*$/d' $1
-  sed -i '/^Nmap done:.*$/d' $1
-  sed -i '/^MAC Address: .* (Unknown)$/d' $1
+  sed -i '/^$/d' "$1"
+  sed -i '/^Starting Nmap.*$/d' "$1"
+  sed -i '/^Nmap scan report for.*$/d' "$1"
+  sed -i '/^Starting Nmap.*$/d' "$1"
+  sed -i '/^Host is up.*$/d' "$1"
+  sed -i '/^Nmap done:.*$/d' "$1"
+  sed -i '/^MAC Address: .* (Unknown)$/d' "$1"
 }
 
 run_nmap () {
-  nmap -T2 --max-parallelism 1 --max-retries 5 --script +ssl-enum-ciphers -p 4343 ${IPADDR} > $1
+  nmap -T2 --max-parallelism 1 --max-retries 5 --script +ssl-enum-ciphers -p 4343 "${IPADDR}" > "$1"
 }
 
-if ${KUBECTL_BINARY} get service -n ${INSTALLED_NAMESPACE} hyperconverged-cluster-webhook-service ; then
+check_ssp_up() {
+  if [[ -n $SSP_DEPLOYED ]]; then
+    ${KUBECTL_BINARY} wait deployment -n "${INSTALLED_NAMESPACE}" ssp-operator --for=condition=Available --timeout="240s"
+  fi
+}
+
+if [[ $(${KUBECTL_BINARY} get ssp -n "${INSTALLED_NAMESPACE}") ]]; then
```

Comment:
and if ssp is not deployed at this point? we continue as usual?

Reply:
also, it is better to actually check if the SSP CR is in the deployed phase, instead of just checking the CR exists. something like:
```suggestion
if [[ $(${KUBECTL_BINARY} get ssp ssp-kubevirt-hyperconverged -n "${INSTALLED_NAMESPACE}" -o jsonpath='{.status.phase}') == "Deployed" ]]; then
```